api/specs/webserver/openapi-computations.yaml

```diff
@@ -1,9 +1,27 @@
 paths:
-  /computation/pipeline/{project_id}/start:
+  /computations/{project_id}:
+    get:
+      description: Returns the last computation data
+      tags:
+        - computations
+      operationId: get_pipeline
+      parameters:
+        - $ref: "#/components/parameters/ProjectId"
+      responses:
+        "200":
+          description: Succesffully retrieved computation
+          content:
+            application/json:
+              schema:
+                $ref: "./components/schemas/pipeline.yaml#/components/schemas/PipelineEnveloped"
```

Comment:
GET on plural nouns refer to List method of collections (https://google.aip.dev/132). Here it is returning a single resource instead ... ! 

A suggestion. First make sure to define what is a "computation" resource.  Define for instance what would you "GET" as a model when you refer to that.

Then try to build the path as *a universal identifier* of that resource (do not think of the path as a filter but as an identifier). For instance, the "user" is a filter criteria (therefore should be in the query) so it should *not be part of the path of the resource (it is not a "permanent" property of the resource since a project could be e.g. shared and then the identifier would loose meaning). Probably the project is part of the identifier so i wonder if some path like GET ``/projects/{project_id}/computations/-/pipeline`` or simply  GET ``/projects/{project_id}/pipeline`` is more suitable for what you want?

Then build standard methods on top of that. And then also define what to return when you use custom methods like start or stop ... In that case, one does not necessary need to return the resource model (or a variant of it) but some returns from specific operations like status, etc ...

Reply:
ok, so I don't understand all the fuss here. First I modified what was already weird: ```computation/pipeline/{project_id}:start``` towards ```computations/{project_id}:start```

So the GET actually get a single resource defined by the project_id. It is true that I could also change the computations endpoint completely to something similar to what the director-v2 does. BUT, this is by far not the focus of this PR and would imply a lot of changes such as the META which seems to be there.
So my plan here is not to change everything but to tend towards that.

Reply:
discussed together